mmdeploy/codebase/base/task.py

```diff
@@ -139,18 +140,38 @@ def single_gpu_test(self,
         return self.codebase_class.single_gpu_test(model, data_loader, show,
                                                    out_dir, **kwargs)
 
+    def reset_model_cfg_norm(self):
+        """Reset the norm values in model_cfg for some backends.
+
+        For example, in rknn-int8 backend, Normalize is moved to rknn models.
+        The processes of visualization and dump-info need this func.
+        """
+        if get_rknn_quantization(self.deploy_cfg):
```

Comment:
What if the user perform backend and pytorch inference in the same thread?

Reply:
Do you mean running the RKNN model through adb remote and pytorch model at the same time? If there is really such a user, `model_cfg` need to be reloaded after normalization set to None.

Reply:
I don't want to add a function for rknn only in BaseTask.
This method could be used to adjust other backends with preprocessing supported. It is possible for OpenVINO user to compare the result of both pytorch and backend.

Reply:
Rename the method to a more **general** name, so we can use it to update pipeline for other backends.
In the future.